src/main/java/be/seeseemelk/mockbukkit/MockUnsafeValues.java

```diff
@@ -296,32 +301,121 @@ public Entity deserializeEntity(byte[] data, World world, boolean preserveUUID)
 		throw new UnimplementedOperationException();
 	}
 
+	/**
+	 * Gets the translation key for a {@link Material} that {@link Material#isBlock() is a block}.
+	 * @param material the material to translate
+	 * @return a string of the structure "block.{@literal <namespace>}.{@literal <material>}" (e.g. "block.minecraft.stone"), or null if not a block.
+	 */
 	@Override
 	public String getBlockTranslationKey(Material material)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		if(!material.isBlock()) {
```

Comment:
Wrong formatting (please reformat all the related files according to mockbukkit specifications; if you're using IntelliJ, this is done automatically)